specification/cognitiveservices/Language.Conversations/tspconfig.yaml

```diff
@@ -12,13 +8,4 @@ options:
     output-file: "azure.ai.language.conversations.json"
     azure-resource-provider-folder: ../../../../data-plane
```

Comment:
Can you reference these standard config values in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerstorage/ContainerStorage.Management/tspconfig.yaml